src/components/Layout/MainStyles.js

```diff
@@ -0,0 +1,109 @@
+import { createGlobalStyle } from 'styled-components';
+
+import { background, blue, foreground, pink } from '@utils/theme';
+
+export const MainStyles = createGlobalStyle`
+html {
+  /* set 1rem to 10px at 1000px wide */
+  font-size: 62.5%;
+  font-size: calc(62.5% - 3px + 0.3vw);
```

Comment:
What is this devilry?

Reply:
Dynamic font sizes! 🎉 

The default font-size for most browsers is `16px`, so it's pretty common to define a default of `font-size: 62.6%`, so that `1rem === 10px`, which then makes using `rem`s a _ton_ more natural.

Adding in `vw` means that the font-size scales to the browser width. So at a few different example screen widths:

320px wide: `1rem` = `10px - 3px + (320 * 0.003)` -> `10px - 3px + .96px` -> ~`8px`
1000px wide: `1rem` = `10px - 3px + (1000 * 0.003)` -> `10px - 3px + 3px` -> `10px`
2000px wide: `1rem` = `10px - 3px + (2000 * 0.003)` -> `10px - 3px + 6px` -> `13px`

That subtle difference in the root em ends up making the entire site scale really nicely to each font size, especially when you have large variance in the font size like in this site. Much cleaner than having to explicitly define media queries with different font sizes for whenever the layout breaks with a static size!

Reply:
Oh, also worth noting the big benefit of using `font-size: 62.5%` over `font-size: 10px` (and by extension, the dynamic version including the percentage): it means that the site is scaled appropriately based on the user's custom default font size.

So as an example, someone has poor eyesight might increase the default font size from `16px` to `22px`. On sites which size everything directly with `px` values, this user setting is completely ignored. On sites styled using `rem` values with the root em based on a percentage value, this user setting is taken into account, and the user sees fonts in a larger size which is easier for them to read!

Reply:
It looks like the responsive text is not respecting the user font size settings on the homepage paragraph and logo as well as the navigation at the top. Is this intentional?

Reply:
Oh damn, good catch - there's a bunch of pixel font sizes I didn't update apparently 🤦‍♂

Will fix that tomorrow!